app/globals.css

```
@@ -6,8 +6,8 @@
 @theme inline {
   --color-background: var(--background);
   --color-foreground: var(--foreground);
-  --font-sans: var(--font-geist-sans);
-  --font-mono: var(--font-geist-mono);
+  --font-sans: DM Sans, sans-serif;
```

Stylelint (17.2.0)

[error] Expected "DM" to be "dm" (value-keyword-case)

(value-keyword-case)